src/Orleans/Core/Grain.cs

```diff
@@ -308,5 +312,20 @@ protected TGrainState State
         {
             get { return base.GrainState as TGrainState; }
         }
+
+        public Task ClearStateAsync()
```

Comment:
Of course you are right. :-) Sorry, I looked at a wrong branch. Currently they need to be public for `GrainState.Read/Write/ClearStateAsync` methods marked obsolete to work. I'll try to rewire those via `Grain.Storage` instead.